### ISSUE TITLE
Bring your own data?

### ISSUE THREAD

**Issue by cthoyt** [AnacletoLAB/grape#6]
I'm emulating a lazy reviewer by only reading the README - I'd much rather bring my own data than use something built in. It would be great to have a example on the README that shows how the data should look (i.e., file format) and also what code you need to use to embed that network

**Comment by LucaCappelletti94** [comment]
Hello @cthoyt and sorry for the long delay, I will be adding that portion of the tutorial in the README. At this time, you can see a long version in [this jupyter notebook](https://github.com/AnacletoLAB/grape/blob/main/tutorials/Loading_a_Graph_in_Ensmallen.ipynb).

I'll be trying to create a super-short version of it.

**Comment by DaminK** [comment]
It would be great if you could also show how to use existing networkx Graph objects. According to the documentation of your GeneralGraph class it should be easy to init it based on an existing nx graph (https://mathlab.github.io/GRAPE/_summaries/grape.general_graph.GeneralGraph.html). However there seems to be a missmatch between your tutorials and the documentation and I can't find out where this class is hidden in the current version.

**Comment by LucaCappelletti94** [comment]
Hi @DaminK, the documentation you have linked has nothing to do with this project. Do join the discord if you need help getting started.

**Comment by DaminK** [comment]
My bad, thanks for you swift reply. However the naming seems really unfortunate to compete with another existing python package for processing graphs.

**Comment by LucaCappelletti94** [comment]
Yes, but it's hardly possible (or desirable) to stop other developers to choose similar names for a project. The important thing in my opinion, for us, is that the python package on pip has a very clear name.
